WordPress/Classes/Extensions/Colors and Styles/UIColor+MurielColors.swift

```diff
@@ -208,6 +208,14 @@ extension UIColor {
         return muriel(color: .divider)
     }
 
+    static var primaryButtonBorder: UIColor {
+           if #available(iOS 13, *) {
+               return .opaqueSeparator
+           }
+
+        return UIColor(displayP3Red: 198.0, green: 198.0, blue: 200.0, alpha: 1.0)
```

Comment:
Wouldn't that be the same as `muriel(color: .gray, .shade10)`?

Reply:
updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/14621/commits/9fc13a87221b9e690eecaa21b7d6018111b2613d